src/middlewares/verifyToken.js

```diff
@@ -7,22 +7,25 @@ const verifyAccessToken = async (req, res, next) => {
   const token = req.header("auth-token");
   if (!token) return onError(res, 401, "Unauthorized");
   try {
-    const verified = jwt.verify(token, process.env.ACCESS_TOKEN_SECRET);
-    client.get(verified.payload.id, (err, val) => {
-      if (verified.payload) {
-        if (val) {
-          req.user = verified.payload;
-          return next();
-        }
-        return onError(
-          res,
-          401,
-          "User already logged out, Please Login and try again!"
-        );
+    await jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (error, decod) => {
+      if (error) {
+        return onError(res, 400, 'Token is incorrect or expired!');
```

Comment:
Could you make it 401 when it is incorrect or expired!

Reply:
ok thanks